package-lock.json

```diff
@@ -5,85 +5,83 @@
   "requires": true,
   "dependencies": {
     "@babel/code-frame": {
-      "version": "7.8.3",
```

Comment:
Probably you deleted `package-lock.json` file, so it got created newly and so all available upgrades to all used npm modules got applied. This also implies upgrade of the related Pods for iOS and libraries for Android. 
Either you revert this change and only add `react-native-zip-archive` without upgrading more modules than necessary or you update the iOS Pods with `cd ios; pod install`.